separate_instance/docker-compose.override.yml

```diff
@@ -0,0 +1,130 @@
+version: "3.4"
+
+services:
+
+#  ########################################################
+#  # FRONTEND #############################################
+#  ########################################################
+#  frontend:
```

Comment:
no need to comment this out you can start stuff specifically `docker-compose up mariadb database backend`

Reply:
yes you're right, but i had trouble with some modul-dependencies, which cause starting modules automatically, but i just not needed them for my scenario.
Commenting them out was the easiest solution...